packages/google_maps_flutter/google_maps_flutter/example/pubspec.yaml

```diff
@@ -19,18 +19,18 @@ dependencies:
     # the parent directory to use the current plugin's version.
     git:
       url: https://github.com/fulpismo/plugins.git
-      ref: v2.2.3-fork.2
+      ref: 'fix_pin_layout'
```

Comment:
what do you think about changing the ref to `v2.2.3-fork.3` so when merged to `main-fork`, you can create a new release named `v2.2.3-fork.3` and all refs will point to the release?